### DISCUSSION TITLE
Inventory Health Report

### DISCUSSION THREAD

**Discussion by chriskshipcor** [amzn/selling-partner-api-models#3220] [General]
I noticed for about the last 6 days the Inventory Health Report has not been reporting 24 Hour sales, which in turn has made every other sales metric in the report incorrect. Is anyone else aware of or also getting this issue? It looks like the order reports and sales api can still see the sales this past week.

**Comment by kepple** [comment]
I spotted the same.  Not only Inventory **Health Report** but also **Restock Inventory Report**, **Manage Inventory Health.** This also affects Inventory Planning UI @ sellercentral. It's probably to do with the firewall outage Amazon recently had.
